config_template.cfg

```diff
@@ -312,12 +312,14 @@ REF_AREA= 1.0
 % Aircraft semi-span (0 implies automatic calculation) (m or in)
 SEMI_SPAN= 0.0
 
-% ---- NONEQUILIBRIUM GAS, IDEAL GAS, POLYTROPIC, VAN DER WAALS AND PENG ROBINSON CONSTANTS -------%
+% ---- NONEQUILIBRIUM GAS, IDEAL GAS, POLYTROPIC, VAN DER WAALS AND PENG ROBINSON CONSTANTS, CoolProp library -------%
 %
 % Fluid model (STANDARD_AIR, IDEAL_GAS, VW_GAS, PR_GAS,
-%              CONSTANT_DENSITY, INC_IDEAL_GAS, INC_IDEAL_GAS_POLY, MUTATIONPP, SU2_NONEQ, FLUID_MIXTURE)
+%              CONSTANT_DENSITY, INC_IDEAL_GAS, INC_IDEAL_GAS_POLY, MUTATIONPP, SU2_NONEQ, FLUID_MIXTURE, COOLPROP)
 FLUID_MODEL= STANDARD_AIR
-%
+% To find all available fluid name for CoolProp library, clikc the following link:
+% http://www.coolprop.org/fluid_properties/PurePseudoPure.html#list-of-fluids
+FLUID_NAME = nitrogen
```

Comment:
@pcarruscag would we able to overload a FLUID_NAME for coolprop and NEMO?   We have GAS_MODEL in NEMO

Reply:
Sure, the two are mutually exclusive, so if you guys are using the gas model as a raw string and not converting to some enum... it should be fine to use the same option.